### PR TITLE
[WIP] feat(pages): update airdrop pages

### LINKED ISSUE #26: Airdrops table styling
## Description

Implement table styling to match the Figma spec.

Subheader copy text is still TBD. Placeholder is fine for now.

## Figma

https://www.figma.com/file/FHGAAxlpvIkCo8qoYlzqqA/Juno-Tools-V1?node-id=20%3A5075

## Screenshot

Screen Shot 2022-02-14 at 1 23 13 PM

### PR DESCRIPTION
**Pull request by grikomsn**
Closes #26 #27 #28

## Description

This PR reworks most airdrop pages and other quality of life improvements.

## Changes

**Airdrops page**

- [x] reworked home page
- [x] reworked airdrops list page (layout, table, search)
- [x] reworked airdrops create page (layout, inputs)

**Misc**

- [x] deprecate theme context
- [x] remove daisyui plugin
- [x] add utility to pass metadatas layout on each page
- [x] fix background stylings
- [x] add react-query

## Screenshots

| / | /airdrops/list |
|---|----------------|
|home|list|

| /airdrops/list search | /airdrops/create |
|-----------------------|------------------|
|search|image|

## Testing Steps

As a reviewer, what steps should I take to verify this is working correctly?

- [x] airdrops list table should render data
- [x] airdrops search input should filter data

## Links

\-

### CODE REVIEW

File: pages/airdrops/create.tsx
Comment (RyanWarner) [inline pages/airdrops/create.tsx:295]: should these inputs be turned into a reusable component?

Comment (grikomsn) [reply pages/airdrops/create.tsx:295]: I haven't updated the create page, will push and request review when it's ready. 👍🏻

**Comment by RyanWarner** [comment]
@grikomsn what is blocking this from being ready for review?

**Comment by grikomsn** [comment]
Will push stylized create airdrop page and updated table stylings, then will mark PR as ready and request reviews.

File: package.json
Comment (RyanWarner) [inline package.json:35]: @findolor are y'all cool with adding this library to the codebase? I personally love it for managing API state, but want to make sure we're all on the same page.

File: pages/airdrops/list.tsx
Comment (RyanWarner) [inline pages/airdrops/list.tsx:209]: This component is getting quite large. Do you see any opportunities to break it up a bit into multiple components?

File: package.json
Comment (grikomsn) [reply package.json:35]: If it's any consolation, `react-datepicker` internally uses `react-popper` dependencies:

https://github.com/CosmosContracts/juno-tools/blob/6dfab36f54d69b582d77e9d49b63f0a88ff5b629/yarn.lock#L4604-L4614

**Review by RyanWarner — APPROVED** [review]
A few comments and questions but overall looking great. Let's give the JunoTools devs some time to review before we merge.

File: pages/airdrops/list.tsx
Comment (grikomsn) [reply pages/airdrops/list.tsx:209]: Sure, I'll break up some parts. Most are still WIP and needed restructuring for #35.

**Review by Highlander-maker — COMMENTED** [review]
That search bar looks slick. Let me have a look when im on my machine in the morning. Loads to go through. 😊.

File: pages/airdrops/list.tsx
Comment (grikomsn) [reply pages/airdrops/list.tsx:87]: Extracted to c4f1476a74ebe976891aedd8654b50a5e7849b4a and ec003a444b567468e23de27b6d37c6d5a2946ad7.

**Comment by grikomsn** [comment]
Pushed the reworked create page stylings. Kept the previous layout until we iterate on an improved UI/UX flow.

**Review by XanderJL — APPROVED** [review]
Only one minor nitpick. Ryan covered other concerns already 👍 😄 👍

**Review by findolor — APPROVED** [review]
LGTM!

Are we going to implement sticky header later on? Either way we are good with merge

File: package.json
Comment (RyanWarner) [reply package.json:35]: @grikomsn this comment was in reference to react-query

**Comment by RyanWarner** [comment]
> Are we going to implement sticky header later on? Either way we are good with merge

yeah sticky header will come next week! #35

**Review by RyanWarner — APPROVED** [review]
fantastic work!

**Pull request merged**
